core/src/components/content/content.tsx

```diff
@@ -183,8 +184,14 @@ export class Content implements ComponentInterface {
    * and `scrollToPoint()` to scroll the content into a certain point.
    */
   @Method()
-  getScrollElement(): Promise<HTMLElement> {
-    return Promise.resolve(this.scrollEl);
+  async getScrollElement(): Promise<HTMLElement> {
+    // if this gets called in certain early lifecycle hooks (ex: Vue onMounted),
+    // scrollEl won't be defined yet, so wait for it to load in
```

Comment:
Super minor style thing, but can we do multi line comments?
```suggestion
    /**
     * If this gets called in certain early lifecycle hooks (ex: Vue onMounted),
     * scrollEl won't be defined yet with the custom elements build, so wait for it to load in.
     */
```